jerry-core/ecma/builtin-objects/ecma-builtin-regexp-prototype.c

```diff
@@ -50,6 +50,30 @@
  */
 
 #if ENABLED (JERRY_ES2015)
+
+/**
+ * Helper function to check if the given value is object
+ *
+ * @return bool
+ */
+static bool
+ecma_op_check_object_type_is_class (ecma_value_t this, /**< this value */
```

Comment:
Could we do this without an output parameter? For example by returning with the extended object pointer or NULL.